integration-tests/start.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+)
+
+type createTokenResponse struct {
+	Token string
+}
+
+type user struct {
+	Email, Password string
+}
+
+type connection struct {
+	Server, Port string
+}
+
+func main() {
+
+	if len(os.Args) == 1 {
```

Comment:
In terms of clean code, it's not good to have a huge block of code inside an if-else clause.
Try to validate if arguments were passed in the beginning and initialize the variables accordingly.
I suggest you to break this code into more methods e get parameters separately as in this example:

```suggestion
  func getUserParams(args) *user {
        usr := user{"test@test.com", "abcdef"}
        if lean(os.Args) == 1 {
          return usr
        }
        
        regex := regexp.MustCompile(`[a-z0-9]*\@[a-z]*\.[a-z]*`)
        switch eval := regex.MatchString(os.Args[1]); eval {
	case true:
		usr.Email = os.Args[1]
	}
	
	switch eval := strings.EqualFold(os.Args[2], ""); eval {
	case false:
		usr.Password = os.Args[2]
	}
	
	return usr
  }
```

Reply:
I will integrate this PR and we can fix this later.